pages/+Wrapper.tsx

```diff
@@ -1,11 +1,36 @@
 import type React from "react";
+import { useEffect, useRef } from "react";
 
-import { fork } from "effector";
-import { Provider } from "effector-react";
+import { EffectorNext } from "@effector/next";
+import { createEvent } from "effector";
+import { useUnit } from "effector-react";
 import { usePageContext } from "vike-react/usePageContext";
 
+const noop = createEvent();
+
+const Inner = () => {
+  const { config } = usePageContext();
+  const clientStartedRef = useRef(false);
+  const onClientStarted = useUnit(config.pageClientStarted ?? noop);
+
+  useEffect(() => {
+    if (!clientStartedRef.current && "pageClientStarted" in config) {
+      onClientStarted();
+      clientStartedRef.current = true;
+    }
+  }, []);
```

Comment:
```suggestion
  }, [onClientStarted]);
```

It is required, when we can switch from page without .pageClientStarted to another with